external-docs/docs/index.md

```diff
@@ -1,17 +1,23 @@
 # Welcome to MkDocs
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+## The Prject
+### What the Project is about
 
-## Commands
+The csvcubed project aims to provide libraries and utilities which make it simple to turn a CSV into 5* linked data. We want to make it easier for data producers to annotate their existing data with the metadata required to ensure that each dataset is discoverable, comparable and analysable by automated tools.
 
-* `mkdocs new [dir-name]` - Create a new project.
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
+The csvcubed catalogue of tools help to deliver this vision in an opinionated fashion. Our tools output data cubes using the CSV on the web (CSV-W) file format. Within these CSV-W files we describe what the data represents using a combination of the following ontologies:
 
-## Project layout
+- RDF Data Cube (qb)
+- Simple Knowledge Organization System (SKOS)
+- Data Catalog Vocabulary (DCAT2)
 
-    mkdocs.yml    # The configuration file.
-    docs/
-        index.md  # The documentation homepage.
-        ...       # Other markdown pages, images and other files.
+### Is this a library or a set of applications?
```

Comment:
I wouldn't include the stuff from here down, it's more targetted at an internal dev audience.